src/pybind/mgr/dashboard/frontend/src/app/shared/api/prometheus.service.ts

```diff
@@ -223,13 +258,35 @@ export class PrometheusService {
               ];
               if (validRangeQueries1.includes(queryName)) {
                 const query = queries[queryName];
-                const request = this.getMultiClusterQueryRangeData({
-                  params: encodeURIComponent(query),
-                  start: selectedTime['start'],
-                  end: selectedTime['end'],
-                  step: selectedTime['step']
-                });
-                requests.push(request);
+                if (
+                  [
+                    'POOL_CAPACITY_UTILIZATION',
+                    'POOL_IOPS_UTILIZATION',
+                    'POOL_THROUGHPUT_UTILIZATION'
+                  ].includes(queryName)
+                ) {
+                  const request = this.getMultiClusterQueryRangeData({
+                    params: encodeURIComponent(query),
+                    start: poolsChartSelectedTime['start'],
+                    end: poolsChartSelectedTime['end'],
+                    step: poolsChartSelectedTime['step']
+                  });
+                  requests.push(request);
+                } else if (
+                  [
+                    'CLUSTER_CAPACITY_UTILIZATION',
+                    'CLUSTER_IOPS_UTILIZATION',
+                    'CLUSTER_THROUGHPUT_UTILIZATION'
+                  ].includes(queryName)
+                ) {
+                  const request = this.getMultiClusterQueryRangeData({
+                    params: encodeURIComponent(query),
+                    start: clusterUtilizationChartSelectedTime['start'],
+                    end: clusterUtilizationChartSelectedTime['end'],
+                    step: clusterUtilizationChartSelectedTime['step']
+                  });
+                  requests.push(request);
+                }
```

Comment:
**THIS IS SERVICE AND IT SHOULD NOT HAVE COMPONENT CODE!**

Reply:
Updated the PR